src/main/java/com/minecolonies/coremod/commands/generalcommands/CheckForAutoDeletesCommand.java

```diff
@@ -0,0 +1,97 @@
+package com.minecolonies.coremod.commands.generalcommands;
+
+import com.minecolonies.api.colony.permissions.Rank;
+import com.minecolonies.api.configuration.Configurations;
+import com.minecolonies.coremod.colony.Colony;
+import com.minecolonies.coremod.colony.ColonyManager;
+import com.minecolonies.coremod.commands.AbstractSingleCommand;
+import net.minecraft.command.CommandException;
+import net.minecraft.command.ICommandSender;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.server.MinecraftServer;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.text.TextComponentString;
+import org.jetbrains.annotations.NotNull;
+
+import javax.annotation.Nullable;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class CheckForAutoDeletesCommand extends AbstractSingleCommand
+{
+
+    public static final  String DESC              = "check";
+    private static final String NO_COLONIES_FOUND = "There were no colonies found that could be auto-deleted";
+    private static final String ABLE_TO_DELETE    = "We were able to delete some colonies!";
+
+    /**
+     * Initialize this SubCommand with it's parents.
+     *
+     * @param parents an array of all the parents.
+     */
+    public CheckForAutoDeletesCommand(@NotNull final String... parents)
+    {
+        super(parents);
+    }
+
+    @Override
+    public boolean canRankUseCommand(@NotNull final Colony colony, @NotNull final EntityPlayer player)
+    {
+        return colony.getPermissions().getRank(player).equals(Rank.OWNER);
+    }
+
+    @Override
+    public void execute(@NotNull final MinecraftServer server, @NotNull final ICommandSender sender, @NotNull final String... args) throws CommandException
+    {
+        final List<Colony> colonies = ColonyManager.getColonies();
+
+        final List<Colony> coloniesToDelete = new ArrayList<>();
+
+        for (int index = 0; colonies.size() - 1 >= index; index++)
+        {
+            final Colony colony = colonies.get(index);
+
+            if (colony.isCanBeAutoDeleted() /*&& Configurations.autoDeleteColoniesInHours != 0*/ && colony.getLastContactInHours()+1 >= Configurations.autoDeleteColoniesInHours)
+            {
+                coloniesToDelete.add(colony);
+            }
+        }
+
+        if (args.length != 0)
+        {
+            if (args[0].equalsIgnoreCase("true"))
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move the "true" string literal on the left side of this string comparison. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1132)